packages/expo-cellular/ios/CellularModule.swift

```diff
@@ -109,6 +109,6 @@ public class CellularModule: Module {
       }
       return providers.values.first
     }
-    return netinfo.subscriberCellularProvider
+    return netinfo.serviceSubscriberCellularProviders?.first?.value
```

Comment:
I understand this would be a slightly bigger change, but reading the code a few lines above that check for iOS 12, could we just assume iOS 12 now and not need to handle this case at all? (Or rather, this line would become something like return nil)

Reply:
Good catch! We can remove this check. I was just looking at the warnings but I will follow up in a couple of more PRs with things like exhaustive switch statements and I will remove these unneeded checks too